.travis.yml

```diff
@@ -59,6 +59,9 @@ matrix:
         - rake --version
         - ruby -S bundle --version
         - ruby -ropen-uri -e 'puts open("https://rubygems.org/") { |f| f.read(1024) }'
+  allow_failures:
```

Comment:
Why do we want this?

Reply:
Otherwise I cannot merge the PR, as I explained in https://github.com/rvm/rvm/pull/5005#issuecomment-729604991.

Reply:
In case it wasn't clear, that job has been failing [for ages](https://travis-ci.org/github/rvm/rvm/builds), and nobody seems to care to fix it, so there is no point to pretend it passes, and I think we can acknowledge it fails and have the CI result actually reflects if the PR breaks something.
Plus, it prevents PRs for contributors like me with write access.

Reply:
Make sense!